File: aicook_api/lib/models/city.js
var mongoose = require('mongoose')

var CitySchema = new mongoose.Schema({
    "index": {
        required: true,
        type: Number
    },
    "dep": {
        required: true,
        type: String
    },
    "name": {
        required: true,
        type: String
    },
    "lat": {
        required: true,
        type: Number
    },
    "long": {
        required: true,
        type: Number
    }
})

var City = mongoose.model('City', CitySchema)
module.exports = {
    City
}

File: aicook_mobile/lib/models/City.dart
class City {
  final String id;
  final String name;
  final String dep;
  final double lat;
  final double lng;

  City({this.id, this.name, this.dep, this.lat, this.lng});

  factory City.fromJson(Map<String, dynamic> json) {
    return City(
        id: json['_id'],
        name: json['name'],
        dep: json['dep'],
        lat: json['lat'].toDouble(),
        lng: json['long'].toDouble());
  }
}
